src/components/BottomBar/index.js

```diff
@@ -29,41 +38,62 @@ class BottomBar extends React.PureComponent {
   }
 
   render() {
-    const { syncBlockNum, syncBlockTime } = this.props;
+    const { classes, syncBlockNum, syncBlockTime } = this.props;
+
+    return (
+      syncBlockTime && syncBlockTime ?
```

Comment:
syncBlockNum && syncBlockTime?

Reply:
this is to hide the bottom bar if data is not loaded